recipes/openpmix/meta.yaml

```diff
@@ -0,0 +1,131 @@
+{% set name = "openpmix" %}
+{% set version = "5.0.4" %}
+
+package:
+  # don't usually want top-level name to match any outputs,
```

Comment:
Comments should be removed from the `meta.yaml` - if there are TODOs to track, open an issue when the feedstock is created to track these.  Other wise the recipe becomes difficult to parse programmatically and contributors have to open the `meta.yaml` to see what needs to be done instead of seeing a separate issue.

Reply:
This is the only place we have to write a todo because the feedstock doesn't exist to open issues, I can remove it once the feedstock is created, since it is only here to workaround a bug in staged recipes, but there's nowhere else to put it for now.

Can I also ask where the idea that all comments should be discouraged in meta.yaml is coming from? I don't think it's true or a good practice and I haven't seen it suggested before. Comments help make clear, maintainable recipes, and should generally be encouraged.

Reply:
Thanks! Hopefully those best practices will encourage code comments for clear, maintainable recipes :)